trialfun/ft_trialfun_brainvision_segmented.m

```diff
@@ -97,18 +97,29 @@
 
 % add the stimulus events to the output, if possible
 numstim = cellfun(@length, stim);
-if all(numstim==numstim(1))
-  for i=1:length(stim)
-    for j=1:numstim(i)
-      stimvalue  = sscanf(value{stim{i}(j)}, cfg.trigformat);
-      stimsample = sample(stim{i}(j));
-      stimtime   = (stimsample - begsample(i) + offset(i))/hdr.Fs; % relative to 'Time 0'
-      trialinfo(i,2*(j-1)+1) = stimvalue;
-      trialinfo(i,2*(j-1)+2) = stimtime;
-    end % j
-  end % i
+if (numstim > 0)
+  if all(numstim==numstim(1))
+    for i=1:length(stim)
+      for j=1:numstim(i)
+        if isempty(value{stim{i}(j)})
+          error('missing a stimulus type definition in the related *.vmrk file');
+        end
+        stimvalue  = sscanf(value{stim{i}(j)}, cfg.trigformat);
+        stimsample = sample(stim{i}(j));
+        stimtime   = (stimsample - begsample(i) + offset(i))/hdr.Fs; % relative to 'Time 0'
+        if isempty(stimvalue)
+          error('the upper case letter of the stimulus value does not match with definition of "cfg.trigformat"'); 
```

Comment:
Of cource, I can do that. Before I want to notice, that I did not add the option 'cfg.trigformat'. This options has been already in the code. I only added a corresponding error message. But  if nobody disturbs, I will make the change and add the option to the doc. At last, just one formal question: After the commit and the push, do I have to force a new pull request? I'm not so familiar with this procedure, since I'm new on GitHub.